Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS_description.md

```diff
@@ -0,0 +1,2 @@
+## CyberArk PAS
```

Comment:
nice. if we are connecting by username and password only, are there any other permissions needed? if so, explain how to add them here.